src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -434,6 +440,8 @@ public float ProgressNotification
         /// <summary>
         /// Gets or sets the properties of the animation.
         /// </summary>
+        [Obsolete("Deprecated in API6, Will be removed in API9, Please use as keyword instead!")]
+        [SuppressMessage("Microsoft.Naming", "CA1819:This API will be deprecated, so suppressing the warning for now")]
```

Comment:
deprecated ACR is required and you have to write a comment based on the current version.